jms-activemq-artemis/src/main/resources/META-INF/native-image/io.micronaut.jms/micronaut-jms-activemq-artemis/reflect-config.json

```diff
@@ -0,0 +1,50 @@
+[
+  {
+    "name": "[Ljava.sql.Date;"
+  },
+  {
+    "name": "[Ljava.sql.Time;"
+  },
+  {
+    "name": "[Ljava.sql.Timestamp;"
+  },
+  {
+    "name": "[Ljava.util.Calendar;"
+  },
+  {
+    "name": "[Ljava.util.Date;"
+  },
+  {
+    "name": "org.apache.activemq.artemis.core.remoting.impl.netty.NettyConnectorFactory",
```

Comment:
Could you please compare with metadata that already exist in the shared repository [here](https://github.com/oracle/graalvm-reachability-metadata/blob/master/metadata/org.apache.activemq/artemis-jms-client/2.28.0/reflect-config.json) and add only that can't be found in the share repo?

Reply:
Thats v2.28.0, we're on 2.31.2 does that make a difference?

Which ones have you seen that need addressing?  Or should I manually go through and check each in turn?

Reply:
There are metadata for `artemis-jms-client` but not for `artemis-jakarta-client` in the shared repository. I was confused because packages and class names are the same. So you need all metadata here until those get added to the shared repo.